sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -266,13 +243,13 @@ def _delay_failure(start_time: datetime.datetime) -> None:
         time.sleep((min_time - (current_time - start_time)).total_seconds())
 
 
-def _get_headers(request_type, **kwargs) -> str:
+def _get_headers(request_type, replica_count, **kwargs) -> str:
```

Comment:
Does it make sense to make this a keyword argument like how we check and evaluate "feature-filters-used" ?

Reply:
So, I have replica_count here as a required field as it will be set in load and refresh. Feature Filters used will be not known at load so it's not a required field.

Reply:
That's fine but it just seemed a bit out of place that we have `replica_count` in the signature of a function to get headers and others are not directly exposed. 

Resolving as this is not blocking.